server/api/lib/trees/regional_tree_permit.js

```diff
@@ -136,50 +141,104 @@ async function saveNewTreePermits(treePermits, maxPermits) {
 	}
 }
 
+const getValueBySheetName = (row, tree_permit_field, sheetName) => {
```

Comment:
let's discuss splitting this to separate crawler modules for the next places we will hopefully crawl :)